docs/intro.rst

```diff
@@ -134,32 +119,78 @@ by importing the package into Python::
 Prerequisite operating system packages
 --------------------------------------
 
-Prerequisite operating system packages can be installed by the pywbem setup
-script using a new command ``install_os``::
+Some of the Python packages used by pywbem have dependencies on operating
+system packages. These operating system packages must be installed before the
+pywbem Python package can be installed. This section describes multiple
+approaches how that can be done, and documents these packages.
+
+* Approach with ``pywbem_os_setup.sh`` script:
+
+  This is the recommended approach, as of pywbem 0.11.
```

Comment:
Suggest it would be clearer if we created subsections for each pywbem release where this differs.  0.11.0, 0.8.1, pre 0.8.1

Reply:
DONE